src/test/ActionsRouter.sol

```diff
@@ -52,11 +53,13 @@ contract ActionsRouter is IUnlockCallback, Test {
             Actions action = actions[i];
             bytes memory param = params[i];
             if (action == Actions.SETTLE) {
-                _settle();
+                _settle(param);
             } else if (action == Actions.SETTLE_NATIVE) {
                 _settleNative(param);
             } else if (action == Actions.TAKE) {
                 _take(param);
+            } else if (action == Actions.TAKE_FROM) {
+                _takeFrom(param);
```

Comment:
i thought for a second youd really added a parameter to take to let you take from someone else 😅 can you rename like `PRANK_TAKE_FROM` or something?